src/Classes/ServiceAPI/MyRadio_List.php

```diff
@@ -67,6 +67,13 @@ class MyRadio_List extends ServiceAPI
     private $members = [];
 
     /**
+     * This is the set of members have opted out of this 'optin' list.
+     *
+     * @var int[]
+     */
+    private $nonMembers = [];
```

Comment:
nonMembers doesn't seem like a very good name for this. Maybe optedOutMembers ? Actually, members, and the SQL it comes from, seems fairly confusing as well. The "mail_subscription::member_id" table/col appears to list the members that have opted out of the mailing list? Very odd

Reply:
I've changed this and added nicer comment. Hopefully this is better now.